event-sauce-derive/src/derives/event_data.rs

```diff
@@ -25,6 +25,7 @@ enum BuilderType {
     Update,
     Delete,
     Purge,
+    Action,
```

Comment:
Uh, the changes tot `event-sauce-derive` should not be here really (included by mistake). It is only my first naive try, which did not work really and needs work. It is not used anywhere in the code, I switched to manually writing the code that ought to be generated by macros (see TODO comments in the action builder test file).